assets/test/specs/addRemove.spec.js

```diff
@@ -54,6 +56,19 @@ describe('Given I have a list of inputs to add/remove', function() {
     it('Delete button should have the default text', function() {
       expect($container3.find('a[data-remove-btn]').text()).toBe('Delete');
     });
+
+    it('Delete buttons should not be present on container 2', function() {
+      expect($container2.find('a[data-remove-btn]').length).toBe(0);
+    });
+
+    it('Delete buttons should be present on container 3', function() {
+      expect($container3.find('a[data-remove-btn]').length).toBe(1);
```

Comment:
@stingrayfunk FYI as a convenience method you can use `.toExist()` for checking that elements exist